plugins/zammad-mcp/skills/zammad-mcp-issue-digest/scripts/test_collect_issue_digest.py

```
@@ -0,0 +1,190 @@
+# mypy: ignore-errors
+"""Regression tests for the Zammad-MCP issue digest collector."""
+
+import importlib.util
+from datetime import datetime, timezone
+from pathlib import Path
+
+SCRIPT_PATH = Path(__file__).with_name("collect_issue_digest.py")
+SPEC = importlib.util.spec_from_file_location("collect_issue_digest", SCRIPT_PATH)
+if SPEC is None or SPEC.loader is None:
+    raise RuntimeError(f"Unable to load collector from {SCRIPT_PATH}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)